.github/workflows/bot.yml

```diff
@@ -545,22 +513,6 @@ jobs:
             flinkProfile: 'flink1.16'
             sparkProfile: 'spark3.3'
             sparkRuntime: 'spark3.3.1'
-          - scalaProfile: 'scala-2.12'
-            flinkProfile: 'flink1.15'
```

Comment:
Move Flink 1.14 and 1.15 validation to existing profiles, and change the docker image in `ci_run.sh`.